apps/web/src/studio/layout/PageContainer.tsx

```diff
@@ -10,7 +10,6 @@ export const PageContainer: FC<PropsWithChildren<IPageContainerProps>> = ({ chil
     <Container
       className={cx(
         css({
-          overflowY: 'auto !important',
```

Comment:
Removing overflow by default. Component consumers now need to explicitly specify when to overflow.